.github/workflows/main_push.yml

```diff
@@ -110,34 +78,29 @@ jobs:
         run: find . -name TEST-*.xml -exec grep -h testcase {} \; | awk -F '"' '{printf("%s#%s() - %.3f s\n", $4, $2, $6); }' | sort -n -k 3 | tail -20
```

Comment:
It's maybe worth to extract this code inside an action to avoid duplicating this complex expression

Reply:
Done in https://github.com/Activiti/Activiti/pull/4529/commits/d464bc01a1c4b968523c5e2b43d9f1f878314bd1